packages/client/src/workflow-client.ts

```diff
@@ -92,12 +103,67 @@ import { mapAsyncIterable } from './iterators-utils';
  *   taskQueue: 'tutorial',
  * });
  * await handle.signal(incrementSignal, 2);
- * await handle.query(getValueQuery); // 4
+ * const queryResult = await handle.query(getValueQuery); // 4
+ * const firstUpdateResult = await handle.executeUpdate(incrementAndGetValueUpdate, { args: [2] }); // 6
+ * const secondUpdateHandle = await handle.startUpdate(incrementAndGetValueUpdate, { args: [2] });
+ * const secondUpdateResult = await secondUpdateHandle.result(); // 8
  * await handle.cancel();
- * await handle.result(); // throws WorkflowExecutionCancelledError
+ * await handle.result(); // throws a WorkflowFailedError with `cause` set to a CancelledFailure.
  * ```
  */
 export interface WorkflowHandle<T extends Workflow = Workflow> extends BaseWorkflowHandle<T> {
+  /**
+   * Start an Update and wait for the result.
+   *
+   * @experimental Update is an experimental feature.
+   *
+   * @throws {@link WorkflowUpdateFailedError} if Update validation fails or if ApplicationFailure is thrown in the Update handler.
+   *
+   * @param def an Update definition as returned from {@link defineUpdate}
+   * @param options Update arguments
+   *
+   * @example
+   * ```ts
+   * const updateResult = await handle.executeUpdate(incrementAndGetValueUpdate, { args: [2] });
+   * ```
+   */
+  executeUpdate<Ret, Args extends [any, ...any[]], Name extends string = string>(
+    def: UpdateDefinition<Ret, Args, Name> | string,
+    options: WorkflowUpdateOptions & { args: Args }
+  ): Promise<Ret>;
+
+  executeUpdate<Ret, Args extends [], Name extends string = string>(
+    def: UpdateDefinition<Ret, Args, Name> | string,
+    options?: WorkflowUpdateOptions & { args?: Args }
+  ): Promise<Ret>;
+
+  /**
+   * Start an Update and receive a handle to the Update.
+   * The Update validator (if present) is run before the handle is returned.
+   *
+   * @experimental Update is an experimental feature.
+   *
+   * @throws {@link WorkflowUpdateFailedError} if Update validation fails.
+   *
+   * @param def an Update definition as returned from {@link defineUpdate}
```

Comment:
`{@link defineUpdate}` doesn't work, but I tried these and they don't work either. Any ideas?

```
workflow#defineUpdate
@temporalio/workflow#defineUpdate
@temporalio/workflow/lib/workflow#defineUpdate
```

https://tsdoc.org/pages/tags/link/

Reply:
I see `defineQuery` and `defineSignal` aren't working either. I don't know why.